fiftyone/core/expressions.py

```diff
@@ -1706,12 +1788,21 @@ def reverse(self):
         """
         return ViewExpression({"$reverseArray": self})
 
-    def sort(self, reverse=False):
-        """Sorts this expression, which must resolve an array whose BSON
-        representation can be sorted by JavaScript's ``.sort()`` method.
+    def sort(self, key=None, reverse=False):
+        """Sorts this expression, which must resolve to an array.
+
+        If no ``field`` is provided, this array must contain elements whose
+        BSON representation can be sorted by JavaScript's ``.sort()`` method.
+
+        If a ``field`` is provided, the array must contain documents, which are
+        sorted by the specified field or embedded field.
```

Comment:
I'm not sure I understand this documentation? `field` refers to the `key` kwarg?